src/DataWrangling/metadata.jl

```diff
@@ -40,6 +40,22 @@ end
 const AnyDateTime = Union{AbstractCFDateTime, Dates.AbstractDateTime}
 const Metadatum   = Metadata{<:AnyDateTime}
 
+"""
+    Metadatum(variable_name;
+                   dataset,
+                   date=first_date(dataset, variable_name),
+                   dir=default_download_folder(dataset))
+
+Specific constructor for a `Metadata` object with a single date, representative of a snapshot in time.
+"""
+function Metadatum(variable_name;
+                   dataset,
+                   date=first_date(dataset, variable_name),
+                   dir=default_download_folder(dataset))
```

Comment:
are we using the word "folder" or "directory"? We should pick one term and use that consistently.

Reply:
I prefer `dir` or directory.
[182cbfb](https://github.com/CliMA/ClimaOcean.jl/pull/400/commits/182cbfb6eae1504803dc67009efe28a065afdf3e)